docs/pkp/intro.md

```diff
@@ -24,9 +24,9 @@ It’s important to note: only those with authorized access have the ability to
 
 ## How do I create a PKP?
 
-You can mint an NFT from our PKP contract on Polygon Mumbai [here](https://explorer.litprotocol.com/mint-pkp). This NFT represents the root ownership of the PKP. The NFT owner can grant other users (via a wallet address) or grant Lit Actions the ability to use the PKP to sign and decrypt data. They also have the ability to assign additional authentication methods, described at the bottom of the page.
+You can mint an NFT from our PKP contract on Chronicle, our OP Stack Rollup chain, [here](https://explorer.litprotocol.com/mint-pkp). This NFT represents the root ownership of the PKP. The NFT owner can grant other users (via a wallet address) or grant Lit Actions the ability to use the PKP to sign and decrypt data. They also have the ability to assign additional authentication methods, described at the bottom of the page.
```

Comment:
"Chronicle, our OP Stack Rollup chain,"

we don't seem to mention elsewhere that Chronicle is an "OP Stack". maybe "Chronicle, Lit's custom rollup testnet" would suffice?

Reply:
yeah good idea, fixed